src/AutoConversionPrompt.js

```diff
@@ -125,65 +125,155 @@ function escapeRegExp(str) {
   return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
 }
 
-function swapSyntax(type) {
-  let pres = document.getElementsByTagName('pre');
-  let usesFakePres = false;
-  if (!pres.length) {
-    pres = document.getElementsByClassName('def'); // why is this so hard?
-    usesFakePres = true;
+function getTypeTable(pre) {
+  const removables = [];
+  const el = pre.nextSibling;
+  if (el && el.classList && el.classList.contains('typetable')) {
+      removables.push(el);
+      let text = maybeTypeTable.innerText;
```

Comment:
Should be `el.innerText`? You can see the error on http://caml.inria.fr/pub/docs/manual-ocaml/libref/Pervasives.html

Really need a test suite or some static analysis in here :) probably should move all this code to Reason soon.